tf/modules/buckets/main.tf

```diff
@@ -94,7 +94,7 @@ resource "google_storage_bucket_iam_member" "backup-upload" {
   bucket = local.gcs_sql_bucket_backup_name
 }
 
-resource "kubernetes_secret" "gcs-hmac-key" {
+resource "kubernetes_secret" "backup-hmac-key" {
   metadata {
     name = "gcs-hmac-key"
```

Comment:
we should probably tweak this too otherwise we have two different resources with the same name just one in TF land and one in k8s land